src/hotspot/share/opto/loopnode.hpp

```diff
@@ -770,6 +771,11 @@ class IdealLoopTree : public ResourceObj {
     return _has_range_checks;
   }
 
+  // Return the strip mined loop tree if there is an outer strip mined loop. Otherwise, return this.
```

Comment:
The comment is not clear for me. Something like next?:
"// Return parent's IdealLoopTree for strip mined loop and `this` for other cases."